--- kdeextragear-3-3.3.2/kdebluetooth/libqobex/qobex/qttylock.cpp.old	2005-03-02 21:57:00.094269160 +0100
+++ kdeextragear-3-3.3.2/kdebluetooth/libqobex/qobex/qttylock.cpp	2005-03-02 21:57:20.716134160 +0100
@@ -177,7 +177,7 @@
 # ifdef HAVE_LOCKDEV_H
     pid_t ret = dev_testlock( QFile::encodeName( device ) );
     if (ret < 0)
-      emit error("Error testing device lock with dev_testlock!");
+//      emit error("Error testing device lock with dev_testlock!");
     return ret;
 # else
     removeStaleLocks();
