--- kdebluetooth-1.0_beta1/kdebluetooth/libqobex/configure.in.in.orig	2006-03-23 09:30:05.373243000 +0100
+++ kdebluetooth-1.0_beta1/kdebluetooth/libqobex/configure.in.in	2006-03-23 09:38:44.763243000 +0100
@@ -398,8 +398,8 @@
 dnl
 
 AC_CHECK_LIB(bluetooth,baswap,BLUETOOTH=1,BLUETOOTH=0)
-AC_CHECK_LIB(bluetooth,sdp_list_free,SDP_IN_BLUETOOTH=1,SDP_IN_BLUETOOTH=0)
-AC_CHECK_LIB(sdp,sdp_list_free,SDP_IN_SDP=1,SDP_IN_SDP=0)
+AC_CHECK_LIB(bluetooth,sdp_list_free,SDP_IN_BLUETOOTH=1,[SDP_IN_BLUETOOTH=0
+AC_CHECK_LIB(sdp,sdp_list_free,SDP_IN_SDP=1,SDP_IN_SDP=0)])
 AC_MSG_CHECKING(for correct SDP lib version)
 AC_TRY_COMPILE([
 #define inline __inline__
--- kdebluetooth-1.0_beta1/kdebluetooth/configure.in.in.orig	2005-03-20 17:46:48.000000000 +0100
+++ kdebluetooth-1.0_beta1/kdebluetooth/configure.in.in	2006-03-23 10:09:59.643243000 +0100
@@ -109,10 +109,10 @@
      AC_MSG_RESULT(no)
   fi
   
-  if test "x$no_bluetooth_2_7" = "x" ; then
-    BLUETOOTH_LIBS=$BLUETOOTH_LIBS_2_7
-  elif test "x$no_bluetooth_2_8" = "x" ; then 
+  if test "x$no_bluetooth_2_8" = "x" ; then
     BLUETOOTH_LIBS=$BLUETOOTH_LIBS_2_8
+  elif test "x$no_bluetooth_2_7" = "x" ; then 
+    BLUETOOTH_LIBS=$BLUETOOTH_LIBS_2_7
   fi
     
   AC_SUBST(BLUETOOTH_CFLAGS)
--- kdebluetooth-1.0_beta1/configure.in.orig	2006-03-23 09:30:05.373243000 +0100
+++ kdebluetooth-1.0_beta1/configure.in	2006-03-23 09:38:44.763243000 +0100
@@ -183,10 +183,10 @@
      AC_MSG_RESULT(no)
   fi
   
-  if test "x$no_bluetooth_2_7" = "x" ; then
-    BLUETOOTH_LIBS=$BLUETOOTH_LIBS_2_7
-  elif test "x$no_bluetooth_2_8" = "x" ; then 
+  if test "x$no_bluetooth_2_8" = "x" ; then
     BLUETOOTH_LIBS=$BLUETOOTH_LIBS_2_8
+  elif test "x$no_bluetooth_2_7" = "x" ; then 
+    BLUETOOTH_LIBS=$BLUETOOTH_LIBS_2_7
   fi
     
   AC_SUBST(BLUETOOTH_CFLAGS)
@@ -398,8 +398,8 @@
 dnl
 
 AC_CHECK_LIB(bluetooth,baswap,BLUETOOTH=1,BLUETOOTH=0)
-AC_CHECK_LIB(bluetooth,sdp_list_free,SDP_IN_BLUETOOTH=1,SDP_IN_BLUETOOTH=0)
-AC_CHECK_LIB(sdp,sdp_list_free,SDP_IN_SDP=1,SDP_IN_SDP=0)
+AC_CHECK_LIB(bluetooth,sdp_list_free,SDP_IN_BLUETOOTH=1,[SDP_IN_BLUETOOTH=0
+AC_CHECK_LIB(sdp,sdp_list_free,SDP_IN_SDP=1,SDP_IN_SDP=0)])
 AC_MSG_CHECKING(for correct SDP lib version)
 AC_TRY_COMPILE([
 #define inline __inline__
