--- kdebluetooth-1.0_beta1/kdebluetooth/kdebluetooth-common/dunhandler/Makefile.am.orig	2005-03-20 17:46:48.000000000 +0100
+++ kdebluetooth-1.0_beta1/kdebluetooth/kdebluetooth-common/dunhandler/Makefile.am	2006-03-21 13:29:29.000000000 +0100
@@ -3,5 +3,5 @@
 dunhandler_SCRIPTS=dunhandler 
 
 # The desktop file
-servicemenudir = $(kde_appsdir)/Utilities
-servicemenu_DATA = dunhandler.desktop
\ Brak znaku nowej linii na koñcu pliku
+servicemenudir = $(kde_appsdir)/.hidden
+servicemenu_DATA = dunhandler.desktop
--- kdebluetooth-1.0_beta1/kdebluetooth/kdebluetooth-common/dunhandler/dunhandler.desktop.in.orig	2005-03-20 17:46:48.000000000 +0100
+++ kdebluetooth-1.0_beta1/kdebluetooth/kdebluetooth-common/dunhandler/dunhandler.desktop.in	2006-03-21 13:32:28.000000000 +0100
@@ -1,10 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
-Name=@dunhandler_dir@/kdebluetooth/dunhandler/dunhandler
-Exec=dunhandler %u
+Name=dunhandler
+Exec=@dunhandler_dir@/kdebluetooth/dunhandler/dunhandler %u
 Icon=network_local
 Type=Application
-Comment=An OBEX Object Push client for the KdeBluetooth Framework
-Comment[et]=OBEX objekti saatmise klient KdeBluetooth raamistikule
+Comment=Dial-Up Networking service handler
 MimeType=bluetooth/dun-profile
-Hidden=true
\ Brak znaku nowej linii na koñcu pliku
--- kdebluetooth-1.0_beta1/kdebluetooth/kdebluetooth-common/faxhandler/Makefile.am.orig	2005-03-20 17:46:48.000000000 +0100
+++ kdebluetooth-1.0_beta1/kdebluetooth/kdebluetooth-common/faxhandler/Makefile.am	2006-03-21 13:13:10.000000000 +0100
@@ -3,5 +3,5 @@
 dunhandler_SCRIPTS=faxhandler kbtfax
 
 # The desktop file
-servicemenudir = $(kde_appsdir)/Utilities
+servicemenudir = $(kde_appsdir)/.hidden
 servicemenu_DATA = faxhandler.desktop
--- kdebluetooth-1.0_beta1/kdebluetooth/kdebluetooth-common/faxhandler/faxhandler.desktop.in.orig	2005-03-20 17:46:48.000000000 +0100
+++ kdebluetooth-1.0_beta1/kdebluetooth/kdebluetooth-common/faxhandler/faxhandler.desktop.in	2006-03-21 13:12:26.000000000 +0100
@@ -1,9 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
-Name=@faxhandler_dir@/kdebluetooth/faxhandler/faxhandler
-Exec=faxhandler %u
+Name=faxhandler
+Name=@faxhandler_dir@/kdebluetooth/faxhandler/faxhandler %u
 Icon=network_local
 Type=Application
 Comment=A handler for the fax profile
 MimeType=bluetooth/fax-profile
-Hidden=true
